### DISCUSSION TITLE
Diminutives

### DISCUSSION THREAD

**Discussion by fisharebest** [fisharebest/gedcom-name#4] [General]
Many people are known by a diminutive form of their given name.  For example, Richard might be known as:

* Rick
* Rich
* Ricky
* Dick

Some diminutives are less obvious, e.g. Margaret -> Peggy.

Individuals may use both full/diminutive versions of the name in different contexts.

**Comment by hartenthaler** [comment]
For me, diminutive name versions are nicknames.

My question is how to store them.

> 1 NAME Margaret /Smith/
> 2 GIVN Margaret
> 2 SURN Smith
> 2 NICK Peggy

or

> 1 NAME Margaret "Peggy" /Smith/
> 2 GIVN Margaret
> 2 SURN Smith
> 2 NICK Peggy

If name pieces are not mandatory in the next version of the GEDCOM standard, only the second version could be used.

**Reply by Norwegian-Sardines** [reply]
OR maybe not use the `NAME.NICK` subtag at all, maybe even remove it from the Standard in v8.0!

**Comment by hartenthaler** [comment]
It will be an implementation thing if a genealogical program knows that "Peggy" is another version of Margaret. So if webtrees offers Soundex search today and maybe in the future an additional diminutive search, that would be nice but isn't a topic for the standard.

**Comment by Norwegian-Sardines** [comment]
While I think current genealogy applications and some users would disagree, I think the following should be used (no `NAME.NICK`) rather than the suggested option.

1 NAME Margaret /Smith/
2 TYPE BIRTH
2 GIVN Margaret
2 SURN Smith
1 NAME Peggy /Smith/
2 TYPE AKA
2 GIVN Peggy
2 SURN Smith

The problem with nicknames and diminutive names is that some of these names are culturally specific and the “concept” does not translate well to everyone. How does an American know that a name is diminutive for another name used in Germany or other country.  How does a modern individual know a name is a diminutive rather than an AKA?

**Comment by fisharebest** [comment]
> It will be an implementation thing if a genealogical program knows that "Peggy" is another version of Margaret.

Some individuals with the name Margaret will use Peggy.  Others may use Maggie, Mags, etc.

So it would be incorrect to assume that every Margaret would call herself Peggy.

> I think the following should be used (no `NAME.NICK`) 

For some individuals, the `NICK` could be an essential part of the name.  e.g. https://en.wikipedia.org/wiki/William_the_Conqueror

```
1 NAME William the Conqueror
2 GIVN William
2 NICK The Conqueror
```

**Reply by Norwegian-Sardines** [reply]
I would never have thought to put “The Conquerer” as a nick name.  I think of it as a descriptive name similar to a location name “of Vinci” for Leonardo de Vinci.  But it would be plausible for someone to do that.

**Comment by fisharebest** [comment]
> I would never have thought to put “The Conquerer” as a nick name.

Interesting.  Perhaps we need a definition of "nickname"?

**Comment by Norwegian-Sardines** [comment]
I think of a nickname as a "Moniker" given to a person to replace their name (either their full name or just their given name). 

"Pretty Boy" Floyd for the bank robber Charles Arthur Floyd 
"The Duke" for Marion Robert Morrison (aka John Wayne)

Rather than a description added on to differentiate a person from another because they have the same name.

But at what point (due to personal or cultural knowledge) does someone determine that there is a difference or some understanding that they have another name?  I dont know!

Erik The Red was given the designation "The Red" to differentiate him from another Erik as I'm sure William was called "The Conqueror" (aka  William the Bastard and other identities) so he could stand apart from his son William Rufus or maybe to be seen as more powerful. 

“The Conqueror” could be seen as being in the same league as a Patronymic where it is just another way of giving a person a differentiating value.  Erik the Red’s son was called Leiv Eiriksson, or Leif the Lucky.
